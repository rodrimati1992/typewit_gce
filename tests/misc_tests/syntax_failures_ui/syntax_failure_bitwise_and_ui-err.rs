
fn main(){
    typewit_gce::gce_int_eq!(3 & 5, 3 & 5, i8);
}


