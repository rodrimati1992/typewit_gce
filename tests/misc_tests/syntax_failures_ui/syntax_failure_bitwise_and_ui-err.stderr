error: this macro doesn't support this syntax
 --> tests/misc_tests/syntax_failures_ui/syntax_failure_bitwise_and_ui-err.rs:3:32
  |
3 |     typewit_gce::gce_int_eq!(3 & 5, 3 & 5, i8);
  |                                ^
  |
  = note: this error originates in the macro `typewit_gce::gce_int_eq` (in Nightly builds, run with -Z macro-backtrace for more info)
